### ISSUE TITLE
SqlStruct::Table#parse! does not support INHERITS operation

### ISSUE THREAD

**Issue by ADone** [SchemaPlus/schema_plus_core#7]
Current regex is catching all chars between first open and last closed brackets, that contains  `INHERITS` operation.
It's caused a problem with schema_plus_foreign_keys at https://github.com/SchemaPlus/schema_plus_foreign_keys/blob/master/lib/schema_plus/foreign_keys/middleware/sql.rb#L12
This code inserts foreign keys after parent tables, not in table's body.
Example:
income sql:
```
CREATE TABLE "account_phones" (
  "id" uuid PRIMARY KEY  DEFAULT uuid_generate_v4(),
  "country_id" uuid NOT NULL
) INHERITS (account_contacts)
```
body after `parse!`:
```
"id" uuid PRIMARY KEY  DEFAULT uuid_generate_v4(), "country_id" uuid NOT NULL ) INHERITS (account_contacts
```
and result of `SchemaPlus::ForeignKeys::Middleware::Sql::Table#after`:
```
CREATE TABLE "account_phones" (
  "id" uuid PRIMARY KEY  DEFAULT uuid_generate_v4(),
  "country_id" uuid NOT NULL
) INHERITS (
  account_contacts,
  CONSTRAINT fk_account_phones_country_id FOREIGN KEY ("country_id") REFERENCES "countries" ("id")
)
```

**Comment by ADone** [comment]
I had tried to fix that, please take a look

**Comment by ronen** [comment]
@ADone thanks again for the fix.  I've merged it and released v0.6.2 of the gem.
